package/lib/src/beam_guard.dart

```diff
@@ -29,7 +29,7 @@ class BeamGuard {
   /// use '/books*'.
   ///
   /// See [_hasMatch] for more details.
-  List<String> pathBlueprints;
+  List<dynamic> pathBlueprints;
```

Comment:
It would be great to update the doc comment for this also, to include `RegExp` and an appropriate example, as is for `String`. Maybe also mention that those are the only 2 types that can be used

Reply:
Yes, I will get to work on documentation it is a priority. I will make sure to point out that we only expect `String` & `RegExp`